crates/api_models/src/health_check.rs

```diff
@@ -1,6 +1,9 @@
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub struct RouterHealthCheckResponse {
-    pub database: String,
-    pub redis: String,
-    pub locker: String,
+    pub database: bool,
+    pub redis: bool,
+    pub locker: bool,
+    pub analytics: bool,
 }
```

Comment:
what about health check for kafka?

Reply:
Will pickup on another PR. Created issue for the same 

https://github.com/juspay/hyperswitch/issues/3501